docs/guides/CODEPEN.md

```diff
@@ -1,73 +1,72 @@
-# Editable Demos in Codepen
+# Editable Demos in CodePen
 
 ## Description
 
-Users will be able to click a button on each demo to open in codepen
-to edit.  From there the user can edit, save or make other
+Users will be able to click a button on each demo to open in CodePen
+to edit. From there the user can edit, save, or make other
 modifications to the example.
 
-## Why Codepen?
+## Why CodePen?
 
-Codepen appears to be one the most stable and active online sandboxes.
-It has less accessibility problems then some of the other tools.
+CodePen appears to be one the most stable and active online sandboxes.
+It also has less accessibility problems than the other tools.
```

Comment:
"less" -> "fewer", though I would reword to say "CodePen offers a more accessible experience compared to other tools"